### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by mindys360**

### CODE REVIEW

**Comment by mindys360** [comment]
Updated to include a total of 20 Cyber terms.

File: cyber/readme.md [hunk 1]
```diff
@@ -4,3 +4,111 @@ _**A**_
    <dd>	A method that is used to confirm the identity of a user, through various means such as password entry or fingerprint scan. This allows them to have access to the system or files.  	</dd>
 </dl>
 
+_**D**_
```
Comment (Laura-Novich-OBW) [inline cyber/readme.md]:
```suggestion
# D
```

Comment (Laura-Novich-OBW) [reply cyber/readme.md]: Please change all of the letters to headings. This way you can link to them from the top of the page.

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]

**Review by mindys360 — COMMENTED** [review]
Updated Glossary to make all letters H1.

File: cyber/readme.md (hunk 1 above)
Comment (mindys360) [reply cyber/readme.md]: Done

**Comment by Laura-Novich-OBW** [comment]
My mistake please make the letters H2s - ## A (for example)
Please also add a title and a short description about the glossary to the top of the page. 
Add your names as contributors and make your names links to your GitHub Profiles
Thank You!

File: cyber/readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Glossary of Cybersecurity Terminology
 **Contributors**
-#### Andrea Parkins
+#### [Andrea Parkins](https://github.com/)
```
Comment (Laura-Novich-OBW) [inline cyber/readme.md]:
```suggestion
#### [Andrea Parkins](https://github.com/akparkins)
```

Comment (mindys360) [reply cyber/readme.md]: File updated with the following:
1. Corrected the link to Github for Andrea Parkins. 
2. Added a sentence below the title.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**